src/css/custom.css

```diff
@@ -6,25 +6,62 @@
 
 /* You can override the default Infima variables here. */
 :root {
-  --ifm-color-primary: #3884ff;
-  --ifm-color-primary-dark: #3884ff;
-  --ifm-color-primary-darker: #3884ff;
-  --ifm-color-primary-darkest: #3884ff;
-  --ifm-color-primary-light: #3884ff;
-  --ifm-color-primary-lighter: #3884ff;
-  --ifm-color-primary-lightest: #3884ff;
+  --ifm-color-primary: #0500e2;
+  --ifm-color-primary-dark: #0500e2;
+  --ifm-color-primary-darker: #0500e2;
+  --ifm-color-primary-darkest: #0500e2;
+  --ifm-color-primary-light: #0500e2;
+  --ifm-color-primary-lighter: #0500e2;
+  --ifm-color-primary-lightest: #0500e2;
   --ifm-code-font-size: 95%;
+  font-family: "Roboto", sans-serif;
+  --ifm-global-radius: 0;
+/*  --ifm-navbar-search-input-background-color: #f5f6f7;*/
+  --ifm-alert-border-width: 5px;
 }
 
 /* For readability concerns, you should choose a lighter palette in dark mode. */
 html[data-theme='dark'] {
-  --ifm-color-primary: #3884ff;
-  --ifm-color-primary-dark: #3884ff;
-  --ifm-color-primary-darker: #3884ff;
-  --ifm-color-primary-darkest: #3884ff;
-  --ifm-color-primary-light: #3884ff;
-  --ifm-color-primary-lighter: #3884ff;
-  --ifm-color-primary-lightest: #3884ff;
+  --ifm-color-primary: #00ffff;
+  --ifm-color-primary-dark: #00ffff;
+  --ifm-color-primary-darker: #00ffff;
+  --ifm-color-primary-darkest: #00ffff;
+  --ifm-color-primary-light: #00ffff;
+  --ifm-color-primary-lighter: #00ffff;
+  --ifm-color-primary-lightest: #00ffff;
+}
+
+/* Custom fonts */
+@font-face {
+  font-family: "Roboto";
+  src: url("/static/fonts/Roboto-Regular.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Roboto Bold";
+  src: url("/static/fonts/Roboto-Bold.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Roboto Mono";
+  src: url("/static/fonts/RobotoMono-Regular.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Artegra Sans Extended";
+  src: url("/static/fonts/ArtegraSansExtended-SemiBold.otf") format("opentype");
```

Comment:
I doubt the license allows us to distribute this in a public repository

Reply:
I did some research. The Artegra studio is the author of the font:
https://artegrastudio.com/index.php/portfolio/artegra-superfamily/

And MyFonts is the official distributor:
https://www.myfonts.com/products/extend-semibold-artegra-sans-234675

The MyFonts licensing for webfonts is here:
https://www.myfonts.com/pages/faq#webfont-licensing

And is explicitly says:
> You will be serving the webfont kit for your own site and linking it in the CSS.

So I think hosting by us is fine as long as the license is there for our docs.ooasis.io. I'll reconsider maybe using a different font though.

Reply:
Hosting it on docs.oasis.io is fine, but not github

Reply:
Nvm, let's ditch the commercial font for now and just go with Roboto (Black) for headings.